Makefile

```diff
@@ -238,6 +240,14 @@ else
   ONLOAD_CFLAGS += -DCI_HAVE_SFC=0
 endif
 
+test-ge = $(shell test $1 -ge $2 && echo 1)
+
+ifneq ($(call test-ge, $(gcc_maj_ver), 10), )
+ifeq ($(KARCH), aarch64)
```

Comment:
We could make the build microscopically faster by turning these conditions around (and making `gcc_maj_ver` delay-evaluated, but I measured it as only ~30ms.

Reply:
It actually seems more logical to check for the arch first and then GCC specifically to apply the arch-specific fix.

Is there anything else to do except this?
```make
gcc_maj_ver = $(shell ./scripts/mmaketool --gcc_major_version)
...
ifeq ($(KARCH), aarch64)
ifneq ($(call test-ge, $(call gcc_maj_ver), 10), )
```

Reply:
I thought there might be other common (non arch specific) things to be set in case of gcc>=10. That's why I put it first.

Reply:
Fair enough, I won't pursue this suggestion as it is a marginal difference.

Thanks for fixing the ARM64 built time.